js_task2/src/js/dataFunctions.js

```diff
@@ -0,0 +1,24 @@
+import { baseUrl, outputArea } from './_variables'
+import axios from 'axios'
+import { renderRow } from './renderFunctions'
+
+export const getUsers = async (baseUrl) => {
```

Comment:
getting and changing data should work without page reloading

Reply:
now works right)